assets/js/script.js

```diff
@@ -0,0 +1,30 @@
+let bookArray = [];
+const html = [];
+const myStorage = window.localStorage;
+bookArray = Object.entries(myStorage);
+
+// eslint-disable-next-line no-unused-vars
+function addBooks() {
+  const bookName = document.getElementById('name-book').value;
+  const bookAuthor = document.getElementById('author-book').value;
+  localStorage.setItem(bookName, bookAuthor);
+}
+
+function removeBooks(i) {
+  const title = i.firstChild.innerText;
+  Object.fromEntries(bookArray);
+  localStorage.removeItem(title);
+  bookArray = Object.entries(myStorage);
+  i.remove();
+}
+
+bookArray.forEach(([key, value]) => {
+  html.push(`<div class='book'><p class='book-title'>${key}</p><p class='book-author'>${value}</p><button class='remove'>Remove</button><br><br></div>`);
```

Comment:
- [ ] OPTIONAL
It will be a good practice to favor the use array methods such as `concat` over `push`, that doesn't mutate the array.